test/schema.rb

```diff
@@ -25,4 +25,17 @@
     t.integer :account_id
     t.integer :blah_id
   end
+
+  drop_table(:fifths) rescue nil
```

Comment:
`force: true` might also work

Reply:
I just copied the other examples in this file.  If you think it should be different, lemme know.